src/Framework/NativeMethods.cs

```diff
@@ -21,9 +24,27 @@
 #nullable disable
 
 namespace Microsoft.Build.Framework;
+
+#if NETFRAMEWORK || NETSTANDARD2_0
+[AttributeUsage(AttributeTargets.Method | AttributeTargets.Property)]
```

Comment:
Any suggestions for where this should go? I kinda just shoved it here to get the fix out.

Reply:
New file here in Framework sounds good to me. Then you can put it in the `System.Runtime.Versioining` namespace and get rid of those ifdefs too :)

Please add doc comments that this is here for compat.